clang-tools-extra/docs/clang-tidy/checks/modernize/use-starts-ends-with.rst

```diff
@@ -3,15 +3,16 @@
 modernize-use-starts-ends-with
 ==============================
 
-Checks whether a ``find`` or ``rfind`` result is compared with 0 and suggests
-replacing with ``starts_with`` when the method exists in the class. Notably,
+Checks for common roundabout ways to express `starts_with` and `ends_with` and
```

Comment:
```suggestion
Checks for common roundabout ways to express ``starts_with`` and ``ends_with`` and
```

Double back-ticks are used for language constructs, single back-ticks - for option names and values.

Reply:
Missed that one -- thanks.